SPEC.md

```diff
@@ -128,6 +129,7 @@ The network configuration is described in JSON form. The configuration can be st
 - `cniVersion` (string): [Semantic Version 2.0](http://semver.org) of CNI specification to which this configuration conforms.
 - `name` (string): Network name. This should be unique across all containers on the host (or other administrative domain).
 - `type` (string): Refers to the filename of the CNI plugin executable.
+- `args` (dictionary): Optional additional arguments provided by the container runtime. For example a dictionary of labels could be passed to CNI plugins by adding them to a labels field under `args`.
```

Comment:
Is there an expectation that `args` will have any correspondence with `CNI_ARGS` in the environment?

Do we need to say which takes precedence if there are matching keys with conflicting values?